server/opts_test.go

```diff
@@ -890,6 +890,94 @@ func TestNkeyUsersConfig(t *testing.T) {
 	}
 }
 
+// Test new nkey users
+func TestTlsWhitelist(t *testing.T) {
```

Comment:
We should probably get in the habit of writing things like `TestTlsAllowList` for new code.

Reply:
name reflects the option tested now